Source/CesiumRuntime/Private/CesiumGltfPointsSceneProxy.cpp

```diff
@@ -0,0 +1,222 @@
+#include "CesiumGltfPointsSceneProxy.h"
+#include "CesiumGltfPointsComponent.h"
+#include "Engine/StaticMesh.h"
+#include "RHIResources.h"
+#include "Runtime/Launch/Resources/Version.h"
+#include "SceneInterface.h"
+#include "StaticMeshResources.h"
+
+#if ENGINE_MAJOR_VERSION > 5 || ENGINE_MINOR_VERSION > 1
```

Comment:
This check seems wrong?

```suggestion
#if ENGINE_MAJOR_VERSION == 5 && ENGINE_MINOR_VERSION > 1
```

Reply:
It's correct. The idea is to use this code in any release greater than 5.2, including a hypothetical future 6.x.

Technically we should check that the version isn't less than 5, but none of this code would compile then anyway so it doesn't much matter.

Reply:
Ah okay, that makes sense.